File: remocra/src/main/java/fr/sdis83/remocra/domain/remocra/RequeteModele.java
package fr.sdis83.remocra.domain.remocra;

import javax.persistence.Column;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

import org.springframework.roo.addon.javabean.RooJavaBean;
import org.springframework.roo.addon.jpa.activerecord.RooJpaActiveRecord;
import org.springframework.roo.addon.json.RooJson;
import org.springframework.roo.addon.tostring.RooToString;

@RooJavaBean
@RooToString
@RooJpaActiveRecord(versionField = "")
@RooJson
public class RequeteModele {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id")
    private Long id;

    private String categorie;

    private String code;

    private String libelle;

    private String description;

    private Boolean spatial;

    private String sourceSql;

}

File: remocra/src/main/webapp/js/app/remocra/model/RequeteModele.js
Ext.define('Sdis.Remocra.model.RequeteModele', {
    extend: 'Ext.data.Model',

    fields: [{
        name: 'id',
        type: 'int',
        useNull: true
    }, {
        name: 'spatial',
        type: 'boolean'
    }, {
        name: 'categorie',
        type: 'string'
    }, {
        name: 'code',
        type: 'string'
    }, {
        name: 'libelle',
        type: 'string'
    }, {
        name: 'description',
        type: 'string'
    }, {
        name: 'sourceSql',
        type: 'string'
    }
    ],
    proxy: {
        type: 'remocra.rest',
        url: Sdis.Remocra.util.Util.withBaseUrl('../requetemodele')
    }
});